src/lib/pages/podcast-gallery/api/load-podcasts.ts

```diff
@@ -5,7 +5,8 @@ export const loadPodcasts: Load = async () => {
   try {
     podcasts.set(await podcastList());
     return {};
-  } catch {
+  } catch (e) {
+    console.error(e);
```

Comment:
Issue: we're exposing stack traces here, let's not do that

Reply:
Why not? What's wrong with exposing the stack trace of a frontend app that's open-source?
And for debugging purposes, at least enable logging during dev

Reply:
OWASP thinks it's a security vulnerability. I don't agree with it either, but for the purposes of this project it's very much to our advantage to conform